src/definitions/ping360.json

```diff
@@ -96,52 +121,73 @@
+                        "maximum": "2",
+                        "default": "0"
                     },
                     {
                         "name": "angle",
                         "type": "u16",
                         "description": "Head angle",
-                        "units": "gradian"
+                        "units": "gradian",
+                        "minimum": "0",
+                        "maximum": "399",
+                        "default": "0"
                     },
                     {
                         "name": "transmit_duration",
                         "type": "u16",
                         "description": "Acoustic transmission duration (1~1000 microseconds)",
-                        "units": "microsecond"
+                        "units": "microsecond",
+                        "minimum": "1",
+                        "maximum": "1000",
+                        "default": "32"
                     },
                     {
                         "name": "sample_period",
                         "type": "u16",
-                        "description": "Time interval between individual signal intensity samples in 25nsec increments (80 to 40000 == 2 microseconds to 1000 microseconds)"
+                        "description": "Time interval between individual signal intensity samples in 25nsec increments (80 to 40000 == 2 microseconds to 1000 microseconds)",
+                        "minimum": "80",
+                        "maximum": "40000",
+                        "default": "80"
                     },
                     {
                         "name": "transmit_frequency",
                         "type": "u16",
                         "description": "Acoustic operating frequency. Frequency range is 500kHz to 1000kHz, however it is only practical to use say 650kHz to 850kHz due to the narrow bandwidth of the acoustic receiver.",
-                        "units": "kHz"
+                        "units": "kHz",
+                        "minimum": "500",
+                        "maximum": "1000",
+                        "default": "740"
                     },
                     {
                         "name": "number_of_samples",
                         "type": "u16",
-                        "description": "Number of samples per reflected signal"
+                        "description": "Number of samples per reflected signal",
+                        "minimum": "200",
+                        "maximum": "1200",
+                        "default": "1024"
                     },
                     {
                         "name": "transmit",
                         "type": "u8",
-                        "description": "0 = do not transmit; 1 = transmit after the transducer has reached the specified angle"
+                        "description": "0 = do not transmit; 1 = transmit after the transducer has reached the specified angle",
+                        "default": "0"
```

Comment:
missing maximum and minimum.

Reply:
This is not a range value. It is an enumeration. I think range does not make sense here.